extend/common-interface/development-branches.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Development branches
+permalink: /extend/common-interface/development-branches/
+---
+
+* TOC
+{:toc}
+
+Development branches are a feature that handles change management in Keboola Connection projects. To learn more about how development branches work refer to the [user documentation](https://help.keboola.com/change-management/). 
+
+{% include branches-beta-warning.html %}
+
+## Running a component in a branch
+
+A component that uses the [Common Interface](/extend/common-interface/) can be run in a branch without any changes to the code. Notable exceptions are components that modify external resources (for example database writers) and components that use injected Storage token to interact with Storage API. 
```

Comment:
inject -> forward 
protože tak se to jmenuje v extend/common-interface/environment/#environment-variables + přidat link na extend/common-interface/environment/#environment-variables